src/test/java/karate/profile/GetProfile.java

```diff
@@ -0,0 +1,10 @@
+package karate.profile;
+
+import com.intuit.karate.junit5.Karate;
+
+public class GetProfile {
+    @Karate.Test
+    Karate testSample() {
+        return Karate.run("classpath:karate/profile/getprofile.feature").relativeTo(getClass());
+    }
+}
```

Comment:
remember you don't need to specify the feature.. why don't you delete this classes? What are you using that for?